config/initializers/friendly_id.rb

```diff
@@ -0,0 +1,96 @@
+# FriendlyId Global Configuration
+#
+# Use this to set up shared configuration options for your entire application.
+# Any of the configuration options shown here can also be applied to single
+# models by passing arguments to the `friendly_id` class method or defining
+# methods in your model.
+#
+# To learn more, check out the guide:
+#
+# http://norman.github.io/friendly_id/file.Guide.html
+
+FriendlyId.defaults do |config|
+  # ## Reserved Words
+  #
+  # Some words could conflict with Rails's routes when used as slugs, or are
+  # undesirable to allow as slugs. Edit this list as needed for your app.
+  config.use :reserved
+
+  config.reserved_words = %w(new edit index session login logout users admin
+    stylesheets assets javascripts images)
+
+  #  ## Friendly Finders
+  #
+  # Uncomment this to use friendly finders in all models. By default, if
+  # you wish to find a record by its friendly id, you must do:
+  #
+  #    MyModel.friendly.find('foo')
+  #
+  # If you uncomment this, you can do:
+  #
+  #    MyModel.find('foo')
+  #
+  # This is significantly more convenient but may not be appropriate for
+  # all applications, so you must explicity opt-in to this behavior. You can
+  # always also configure it on a per-model basis if you prefer.
+  #
+  # Something else to consider is that using the :finders addon boosts
+  # performance because it will avoid Rails-internal code that makes runtime
+  # calls to `Module.extend`.
+  #
+  # config.use :finders
+  #
+  # ## Slugs
+  #
+  # Most applications will use the :slugged module everywhere. If you wish
+  # to do so, uncomment the following line.
+  #
+  # config.use :slugged
+  #
+  # By default, FriendlyId's :slugged addon expects the slug column to be named
+  # 'slug', but you can change it if you wish.
+  #
+  # config.slug_column = 'slug'
```

Comment:
ah hah!! This answers our question about the slug column  (remember we were wondering if we actually needed it or not since the readme didn't mention to add one) :smile:  Looks like FriendlyId needs it, but you could rename it to something else if here if you wanted by commenting in this line and doing `config.slug_column = "hamster"` and then chance your database column to be `hamster` via a migration, and tada! hamsters statt slugs. (I'm not suggesting you do this, this is just an example & an interesting configuration option).

Reply:
Thanks a lot Tam! I might change "slug" to sth more understandable like "pretty_url" then, in the `config/initializers/friendly_id.rb` as well as in the `db`. Yeah :tada:

Reply:
Actually, I changed my mind, I wasn't sure abt the easyness of renaming the "slug" column, bc there is this second migration, that would probably needed to be changed, too...